physics/sfcsub.F

```diff
@@ -7394,6 +7546,7 @@ subroutine clima(lugb,iy,im,id,ih,fh,len,lsoil,slmskl,slmskw,     &
         endif
 !
 !  soil type
+        print *,'in FIXREAD fnsotc =',fnsotc 
```

Comment:
same comment about print lines

Reply:
Although some of the comments were referring to pre-existing aborts, most of the comments refer to new print statements added in this PR. People complain about command line output being "polluted". I understand the developer need to have print lines in the code, but they should not be active by default. @GeorgeGayno-NOAA what do you think?

Reply:
Yes, you can get rid of the print line.